src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Debugging/DefaultCSharpBreakpointResolver.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Reflection;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Debugging
+{
+    // This type is temporary and will be replaced by an ExternalAccess.Razor type once available.
+
+    [Export(typeof(CSharpBreakpointResolver))]
+    internal class DefaultCSharpBreakpointResolver : CSharpBreakpointResolver
+    {
+        private readonly MethodInfo _tryGetBreakpointSpan;
+
+        public DefaultCSharpBreakpointResolver()
+        {
+            try
+            {
+                var assemblyName = typeof(SyntaxTree).Assembly.GetName();
+                assemblyName.Name = "Microsoft.CodeAnalysis.CSharp.Features";
```

Comment:
That is a big ouch on the workaround here. But I also happen to know that the API hamstrings you so there's probably not a better way.

Reply:
Ya unfortunately no better way.